React/wbooks/src/Home.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
```

Comment:
All components that don't have state or instance methods should be functional stateless components: 

```
function MyStatelessComponent({ prop1, prop2, ... }) {´
  return (
    /// JSX
  )
}

export default MyStatelessComponent
```